### PR TITLE
docs: improve changelog for Inactive Provider Validators

### PR DESCRIPTION
**Pull request by mpoke**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct `docs:` prefix in the PR title
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] reviewed "Files changed" and left comments if necessary 
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] Confirmed the correct `docs:` prefix in the PR title
- [ ] Confirmed all author checklist items have been addressed 
- [ ] Confirmed that this PR only changes documentation
- [ ] Reviewed content for consistency
- [ ] Reviewed content for spelling and grammar
- [ ] Tested instructions (if applicable)
- [ ] Checked that the documentation website can be built and deployed successfully (run `make build-docs`)

## Summary by CodeRabbit

- **New Features**
	- Introduced the _Inactive Provider Validators_ feature, enhancing provider functionality within the consensus mechanism.
	- Added a new `min_stake` configuration parameter for consumer chains to set minimum stake requirements for validators.
	- Launched the `allow_inactive_vals` option, providing flexibility in validator management by enabling inactive validators to participate in processes.

### CODE REVIEW

**Review by p-offtermatt — APPROVED** [review]
LGTM, thank you!

File: .changelog/unreleased/api-breaking/provider/2079-inactive-validators.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+- Add the _Inactive Provider Validators_ feature (as per 
+  [ADR-017](https://cosmos.github.io/interchain-security/adrs/adr-017-allowing-inactive-validators)),
+  which entails the following changes on the provider.
+  ([\#2079](https://github.com/cosmos/interchain-security/pull/2079))
+
+  - Add `max_provider_consensus_validators`, a provider module param that sets 
+    the maximum number of validators that will be passed to the provider consensus engine.
+  - Add `no_valupdates_genutil` and `no_valupdates_staking`, "wrapper" modules around 
+    the Cosmos SDK's native genutil and staking modules. Both modules provide the exact 
+    same functionality as the native modules, except returning validator set updates
```
Comment (insumity) [inline .changelog/unreleased/api-breaking/provider/2079-inactive-validators.md:10]:
```suggestion
    same functionality as the native modules, except NOT returning validator set updates 
```
?

File: .changelog/unreleased/features/provider/2035-min-stake-max-rank.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
-- Add minimum stake requirement to let consumer chains
-  determine requirements for validators that validate their chain. ([\#2035](https://github.com/cosmos/interchain-security/pull/2035))
+- Add `min_stake`, a power shaping configuration that enables consumers to set
```
Comment (insumity) [inline .changelog/unreleased/features/provider/2035-min-stake-max-rank.md:1]: Mention that `min_stake` this is a parameter?
Similarly below for `allow_inactive_vals`.

Comment (p-offtermatt) [reply .changelog/unreleased/features/provider/2035-min-stake-max-rank.md:1]: imo this is clear from the context
anyways, cheap to add, so I did

**Comment by p-offtermatt** [comment]
Thanks @insumity, since @mpoke is out I incorporated the comments

**Review by insumity — APPROVED** [review]

**Pull request merged**